lightkurve/tests/test_lightcurve.py

```diff
@@ -55,9 +55,14 @@ def test_bitmasking(quality_bitmask, answer):
 
 def test_lightcurve_fold():
     """Test the ``LightCurve.fold()`` method."""
-    lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])
-    assert_almost_equal(lc.fold(period=1).time[0], 0)
-    assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)
+    lcf = KeplerLightCurveFile(TABBY_Q8, quality_bitmask=None)
+    lc = lcf.PDCSAP_FLUX.remove_nans()
+    fold = lc.fold(period=1)
+    assert_almost_equal(fold.time[0], -0.5, 2)
+    fold = lc.fold(period=1, phase=-0.1)
+    assert_almost_equal(np.min(fold.time), -0.5, 2)
+    assert_almost_equal(np.max(fold.time), 0.5, 2)
+    assert_almost_equal(fold.time[np.argmin(fold.flux)], 0.319, 2)
```

Comment:
it's not clear to me where  ``0.319`` came from.

I quite like the simple test before, i.e., it was clear to me that 
``lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])``.

Is it possible to, in addition to this test you've added, to keep the old one, but correct it according to the changes you've made?

Reply:
Let's keep the old test as well. It does not hurt, and it's more intuitive.

Reply:
Also, can you add this note you've written about ``0.319`` as a comment in the code?

Reply:
I'm actually removing the test I've changed my mind I don't like it.